content/concepts/basics/04-search.md

```diff
@@ -7,11 +7,41 @@ metaDescription: "sensenet Search"
 # Search
 https://community.sensenet.com/docs/search-engine/
 https://wiki.sensenet.com/query_syntax.html
-# Full text search
+
+In sensenet one of the most important technical aspects is how we index content items and how we search for them. Especially because sensenet is a query based system which means every content, every list you see on the UI is there because a query found it. In this article we describe the search module and how developers can customize it.
+
 # Metadata indexing, searching by fields
-https://wiki.sensenet.com/Field_Indexing.html
+Making full index of content repository content is one of the most important tasks of the installation procedure of the sensenet Content Repository. This feature is used frequently during development too. The tool that can be used to create a full index is the Index Populator.
```

Comment:
I don't think we should highlight installation procedure here because I don't think it is tied up to it.